spritzle/main.py

```diff
@@ -62,7 +62,23 @@
                  spritzle.resource.auth.auth_middleware])
 
 
-def setup_routes():
+def setup_app(app, core):
+    config = core.config
+    if not config['auth_secret']:
```

Comment:
There is a bunch of redundant code in main() still, please take a look and clean it up.